firedrake/functionspace.py

```diff
@@ -135,6 +135,45 @@ def FunctionSpace(mesh, family, degree=None, name=None, vfamily=None,
         return new
 
 
+@timed_function("CreateSpace")
+def DualSpace(mesh, family, degree=None, name=None, vfamily=None,
```

Comment:
Do we need this constructor or should one just construct dual spaces with `Vstar = FunctionSpace(...).dual()` ?